### DISCUSSION TITLE
Does --reverse-only param still work for import process ?

### DISCUSSION THREAD

**Discussion by felix-geovelo** [osm-search/Nominatim#3381] [Q&A]
**Describe the bug**
I can't see any difference in the import process with and without --reverse-only. In both case I have the same logs output.

**To Reproduce**
I tried to import a small PBF file with and without the --reverse-only param to see the difference in the import logs. But there is apparently none.
I was expecting that with --reverse-only, all logs looking like "Rank ... ETA" would disappear but that was not the case (i.e that all indexing stage would be skipped)
Does this feature still works ? Is it supposed to save a lot of time during the import process for large PBF file ?

**Software Environment (please complete the following information):**
- Nominatim version: 4.4.0
- Postgresql version: 16
- Postgis version: 3.4.2
- OS: Ubuntu 22.04.4 LTS

**Hardware Configuration (please complete the following information):**
- RAM: 126 Go
- number of CPUs: 16
- type and size of disks: 2T of NVME

**Postgresql Configuration:**

RAS

**Nominatim Configuration:**

Variable set in .env :
NOMINATIM_DATABASE_DSN & NOMINATIM_DATABASE_WEBUSER

**More info:**

I'm trying to create a nominatim DB to import into photon after, so --reverse-only could help me to save some time during import process by skipping indexing stages

**Comment by mtmail** [comment]
Indexing by rank is still needed for a reverse-only database. It calcuzwhich streets belong to which city, which city to which state etc. Only the search index creation is skipped. That’s maybe 10% of total import time.

**Comment by felix-geovelo** [comment]
Thanks for the quick and clear reply, maybe it could be a useful add to the doc saying that --reverse-only save up to 10% of import time on behalf of 5% disk space.

Thanks again
